SConstruct

```diff
@@ -92,6 +92,8 @@ def parse_and_save_conf(env, opts_file):
     opts.Add(EnumVariable('SCONS_ENV', "Default SCons environment inheritance",
                           'minimal', ['minimal', 'full'], ignorecase=2))
 
+    opts.Add(BoolVariable('STATIC_FUSE', "Build with static libfuse library", 0))
```

Comment:
This should probably go in https://github.com/daos-stack/daos/blob/master/site_scons/prereq_tools/base.py#L400

We moved all the options that we could to that helper file, the only ones in the top level SConstruct file are ones that are needed to bootstrap the build process.

Reply:
prefer to handle in a subsequent patch.